packages/button/src/vaadin-button.js

```diff
@@ -43,15 +43,23 @@ class Button extends ControlStateMixin(ActiveMixin(ElementMixin(ThemableMixin(Po
           display: inline-flex;
           align-items: center;
           justify-content: center;
-          text-align: center;
+          padding: 0;
           width: 100%;
           height: 100%;
           min-height: inherit;
-          text-shadow: inherit;
           background: transparent;
-          padding: 0;
-          border: none;
```

Comment:
Let's revert some changes like these two lines and also the `text-align: center` and `text-shadow` above (those that appear to be only related to CSS properties reordering) to make the diff smaller.

Reply:
Done.